robotiq_ft_driver/src/robotiq_ft_driver_node.ros2.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <memory>
+
+#include <robotiq_ft_driver/robotiq_ft_driver.hpp>
+
+// ROS
+#include <rclcpp/rclcpp.hpp>
+#include <geometry_msgs/msg/wrench_stamped.hpp>
+
+namespace robotiq_ft_driver
+{
+class RobotiqFTDriverNode : public rclcpp::Node
+{
+private:
+  std::string sensor_frame_;
+  using WrenchStamped = geometry_msgs::msg::WrenchStamped;
```

Comment:
I think it would be a bit clearer if this were defined outside of the `RobotiqFTDriverNode` in this files, rather than inline with the private members.

Reply:
Hmm, I'd rather not pollute the namespace with type aliases whose sole purposes is to keep line length within reasonable bounds.

See https://github.com/ToyotaResearchInstitute/tri_hardware_drivers/commit/a5df857293235f5f7fe58d1400bf9eaaabfd6f17